playbooks/common/openshift-cluster/upgrades/pre/verify_cluster.yml

```diff
@@ -41,6 +41,16 @@
     - openshift_release is defined
     - not (openshift_release is version_compare(openshift_upgrade_target ,'='))
 
+  - fail:
+      msg: >
+        No schedulable masters found, please make sure at least one master doesn't have 'openshift_schedulable=false'
+    when:
```

Comment:
I'm not sure this is adequate.  We forced the masters to scheduleable in 3.9; so there may indeed be schedulable masters (without the variable set) and one or two with scheduling disabled in inventory.

Additionally, someone may have changed the schedulability manually via the oc command and not updated the inventory.

Probably best to query the cluster for this info.

In any case, I thought we had to have masters schedulable now, I'm confused on why we should need this.